packages/gp2-components/src/templates/UsersPage.tsx

```diff
@@ -45,6 +88,22 @@ const UsersPage: React.FC<UsersPageProps> = ({
   const { users } = gp2;
   const backHref = users({}).$;
   const onBackClick = () => changeLocation(backHref);
+
+  const onRemove = useCallback(
+    (id: FilterPillId, typeOfFilter: FiltersType) => {
+      const newFilter = (filters[typeOfFilter] || []).filter(
```

Comment:
we can do it directly, I think the name newFilter here is a bit confusing since we are removing